src/client/Licit.js

```diff
@@ -630,6 +634,27 @@ class Licit extends React.Component<any, any> {
     });
   };
 
+  exportJSON = () => {
```

Comment:
This function in unnecessary. To call it, the caller will already have access to the doc and the export plugin. For type safety, the export function should be accessed directly.